WWDCFamily/Source/Auth/Controllers/AuthViewController.swift

```diff
@@ -25,10 +32,36 @@ class AuthViewController: UIViewController {
     override var preferredStatusBarStyle: UIStatusBarStyle {
         return .lightContent
     }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        self.authStateDidChangeHandle = self.auth?.addStateDidChangeListener(self.updateUI(auth:user:))
```

Comment:
I think that's a good idea, gonna include it in a future PR, moved the rest of the changes to use the router, though.